### PR TITLE
Feat/add books filter query

### PR DESCRIPTION
**Pull request by sagar-wednesday**
### Ticket Link

---

### Related Links

---

### Description

---

### Steps to Reproduce / Test

---

---

### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added]
- [ ] Relevant documentation is changed or added (and PR referenced)

### GIF's

---

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -37,7 +37,7 @@ module.exports = {
     'newline-per-chained-call': 0,
     'no-confusing-arrow': 0,
     'max-lines': ['error', { max: 300, skipBlankLines: true, skipComments: true }],
-    'no-unused-vars': 2,
+    'no-unused-vars': 1,
```
Comment (alichherawalla) [inline .eslintrc.js]: what is the intent of changing this?

Comment (sagar-wednesday) [reply .eslintrc.js]: so that the unused vars should be warning in place of error sign

**Review by alichherawalla — APPROVED** [review]

File: .eslintrc.js (hunk 1 above)
Comment (alichherawalla) [reply .eslintrc.js]: ah got it. It should be an error though right? 
If the variable is unused we don't need it in the codebase

Comment (sagar-wednesday) [reply .eslintrc.js]: yeah

Comment (sagar-wednesday) [reply .eslintrc.js]: but during development we should be able to differ between the actual errors and the errors from the unused vars

**Review by alichherawalla — APPROVED** [review]

**Pull request merged**